src/webview/wkwebview/mod.rs

```diff
@@ -584,6 +584,13 @@ r#"Object.defineProperty(window, 'ipc', {
       (view_frame.size.width as f64, view_frame.size.height as f64).into();
     logical.to_physical(scale_factor)
   }
+
+  pub fn zoom(&self, scale_factor: f64) {
+    #[cfg(target_os = "macos")]
+    unsafe {
+      let _: () = msg_send![self.webview, setPageZoom: scale_factor];
```

Comment:
According to https://developer.apple.com/documentation/webkit/wkwebview/3516411-pagezoom?language=objc , this api seems to be available on iOS 14+. And is only available in macOS 11+, should we document that @wusyong?

Reply:
Yes, please add it platform specific doc.